testsuites/CBLTester/System_test_multiple_device/test_p2p_custom_conflict_resolution.py

```diff
@@ -0,0 +1,487 @@
+import pytest
+
+from CBLClient.Replication import Replication
+from CBLClient.PeerToPeer import PeerToPeer
+from keywords.utils import random_string, log_info
+from testsuites.CBLTester.CBL_Functional_tests.TestSetup_FunctionalTests.test_delta_sync import property_updater
+
+
+@pytest.mark.sanity
+@pytest.mark.listener
+@pytest.mark.custom_conflict
+@pytest.mark.replication
+@pytest.mark.parametrize("replicator_type, endpoint_type", [
+    ("pull", "URLEndPoint"),
+    ("pull", "MessageEndPoint"),
+    ("push_pull", "URLEndPoint"),
+    ("push_pull", "MessageEndPoint")
+])
+def test_p2p_local_wins_custom_conflicts(params_from_base_test_setup, server_setup, replicator_type, endpoint_type):
+    """
+    @summary: resolve conflicts as per local doc
```

Comment:
please add same summary as spreadsheet  . Please do it for all tests

Reply:
see previous comments